syne_tune/optimizer/schedulers/searchers/bore/bore.py

```diff
@@ -216,7 +216,14 @@ def _train_model(self, train_data: list, train_targets: list) -> bool:
         y = np.array(train_targets)
 
         tau = np.quantile(y, q=self.gamma)
-        z = np.less(y, tau)
+        z = np.less_equal(y, tau)
+        if np.sum(z) == y.shape[0]:
+            logging.error(
```

Comment:
This should be a warning right?

```suggestion
            logging.warning(
```